docs/openapi.yaml

```diff
@@ -374,7 +447,19 @@ components:
           example: 99
           minimum: 0
           maximum: 100
-  
+
+    BindApp:
+      type: object
+      properties:
+        app-name:
+          type: string
+        app-host:
+          type: string
+        user:
+          type: string
+        eventid:
+          type: string
```

Comment:
Can you add the missing fields `app-hosts` and `app-internal-hosts`? The later was included on https://github.com/tsuru/tsuru/pull/2442... there you can find some description about.

Providing some [examples](https://swagger.io/docs/specification/adding-examples/) can help to improve this spec.